src/main/scala/com.snowplowanalytics.sauna/config/RespondersConfig.scala

```diff
@@ -36,7 +36,9 @@ object RespondersConfig {
   def apply(respondersDirectory: String): RespondersConfig = {
     val optimizelyFile = new File(s"$respondersDirectory/optimizely_config.json")
     val sendgridFile = new File(s"$respondersDirectory/sendgrid_config.json")
-
+    val urbanairshipFile = new File(s"$respondersDirectory/urban_airship_config.json")
+    val mailchimpFile = new File(s"$respondersDirectory/mailchimp_config.json")
```

Comment:
Please, keep all code related to MailChimp in a separate branch, so we could concentrate our work on a single responder.